docs/cloud-tutorials/best-practices.md

```diff
@@ -0,0 +1,87 @@
+---
+sidebar_position: 8
+title: Best practices
+description: Tips and tricks for succesfully using DBOS Cloud
+---
+
+This section dives into specific behaviors and corner cases you might encounter when using DBOS Cloud.
+
+## Managing applications
+
+### Registration
+* An application must be registered before being deployed. DBOS Cloud CLI will handle registration by default when you deploy an application for the first time.
```

Comment:
We won't need this soon.

Reply:
I think we should keep it around and mention its done automatically for you.